layers/chirpstack/meta-chirpstack/recipes-chirpstack/chirpstack-network-server/chirpstack-network-server/config/as923.toml

```diff
@@ -0,0 +1,12 @@
+[general]
```

Comment:
You probably want to add some `extra_channels` configuration, like with the EU868 configuration. As a device will start with two channels configured, the network-server must communicate the additional channels to the device.

https://github.com/brocaar/chirpstack-gateway-os/blob/master/layers/chirpstack/meta-chirpstack/recipes-chirpstack/chirpstack-network-server/chirpstack-network-server/config/eu868.toml#L14

This list would cover the following channels (as this list contains the extra channels added to the pre-defined channels):

  923600000
  923800000
  924000000
  924200000
  924400000
  924600000

Reply:
@brocaar good point. I've added the extra channels.